model/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -85,8 +85,11 @@
         
         <exclude-unlisted-classes>true</exclude-unlisted-classes>
 
+        <!-- These properties are overridden when Keycloak is managing the persistence unit by itself -->
         <properties>
             <property name="jboss.as.jpa.managed" value="false"/>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
+            <property name="hibernate.query.startup_check" value="false"/>
```

Comment:
?

Reply:
I can make the `hibernate.query.startup_check` configurable. This also saves some time during startup given that validating named queries is expensive for us.